airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/cdc/PostgresCdcCtidInitializer.java

```diff
@@ -244,49 +257,101 @@ public static List<AutoCloseableIterator<AirbyteMessage>> cdcCtidIteratorsCombin
     final AirbyteDebeziumHandler<Long> handler = new AirbyteDebeziumHandler<>(sourceConfig,
         targetPosition, false, firstRecordWaitTime, queueSize, false);
     final PostgresCdcStateHandler postgresCdcStateHandler = new PostgresCdcStateHandler(stateManager);
-    final var cdcStreamList = catalog.getStreams().stream()
+    final var allCdcStreamList = catalog.getStreams().stream()
         .filter(stream -> stream.getSyncMode() == SyncMode.INCREMENTAL)
         .map(stream -> stream.getStream().getNamespace() + "." + stream.getStream().getName()).toList();
-    final var propertiesManager = new RelationalDbDebeziumPropertiesManager(
-        PostgresCdcProperties.getDebeziumDefaultProperties(database), sourceConfig, catalog, cdcStreamList);
-    final var eventConverter = new RelationalDbDebeziumEventConverter(new PostgresCdcConnectorMetadataInjector(), emittedAt);
+    // Debezium is started for streams that have been started - that is they have been partially or
```

Comment:
nit and optional: can you also note this is for incremental streams? (better yet I think you can put following logic into a separate function so it's pure wass operation (incremental streams only) so it doesn't confuse readers with full refresh)

Reply:
I've added the comment. However, we need to figure out the started cdc stream list here so that we can figure out in line 288 which of the scenarios the sync falls under : 
1. First sync (no prev state) : in which case we do initial sync + CDC
2. Purely incremental sync
3. WASS case